llvm/lib/Target/RISCV/RISCVISelDAGToDAG.cpp

```diff
@@ -2351,7 +2357,8 @@ bool RISCVDAGToDAGISel::SelectAddrRegRegScale(SDValue Addr,
 }
 
 bool RISCVDAGToDAGISel::SelectAddrRegImm(SDValue Addr, SDValue &Base,
-                                         SDValue &Offset, bool IsINX) {
+                                         SDValue &Offset, bool IsINX,
+                                         bool IsPrefetch) {
   if (SelectAddrFrameIndex(Addr, Base, Offset))
     return true;
 
```

Comment:
Looks like the code below allows an ADD_LO to be matched, but you can't guarantee the final immediate has 0s in the least significant bits.